airbyte-integrations/connectors/source-zendesk-talk/metadata.yaml

```diff
@@ -11,7 +11,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: c8630570-086d-4a40-99ae-ea5b18673071
-  dockerImageTag: 0.1.13
+  dockerImageTag: 0.1.14
```

Comment:
can we instead make this a minor version bump since this an additive feature to add record counts

Reply:
Done!